content/en/users/aai/check-in/service-accounts/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Service Accounts
+linkTitle: Service Accounts
+type: docs
+weight: 50
+description: >
+  Create a Service Account using EGI Check-in
+---
+
```

Comment:
I would first describe what a service account is and why it would be used.

Reply:
I just added a brief intro, please check

Reply:
Thanks, looks better! We still leave too much room for confusion though. We should provide _specific_ scenarios - a concrete example (maybe the one in the business requirement).